plugins/debandshit/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, vapoursynthPlugins, buildPythonPackage, fetchFromGitHub, python3, vapoursynth }:
+let
+  propagatedBinaryPlugins = with vapoursynthPlugins; [
+    f3kdb
+    fmtconv
+    #rgsf
+  ];
+in
+buildPythonPackage rec {
+  pname = "debandshit";
+  version = "unstable-2020-05-28";
+
+  src = fetchFromGitHub {
+    owner = "LightArrowsEXE";
+    repo = pname;
+    rev = "d5529c97cdd7f01dcc261d6e79b2aa63b3758cbe";
+    sha256 = "0gwpjglm74y6a7x9ws7is7algnwm3i0ds65nz3ki8faigx1jh52a";
+  };
+
+  propagatedBuildInputs = (with vapoursynthPlugins; [
+    fvsfunc
+    muvsfunc
+    mvsfunc
+  ]) ++ propagatedBinaryPlugins;
+
+  format = "other";
+
+  installPhase = ''
+    runHook preInstall
+    install -D debandshit.py $out/${python3.sitePackages}/debandshit.py
+    runHook postInstall
+  '';
+
+  checkInputs = [ (vapoursynth.withPlugins propagatedBinaryPlugins) ];
```

Comment:
Hmm, I get an error with `withPlugins` not existing on `vapoursynth` here, and I think it might be related to the plugins using the Python `callPackage` instead of the main one.

Reply:
Hmm… I can’t reproduce that, neither with the nixpkgs version from the flake, nor the latest unstable (`540dccb2aeaffa9dc69bfdc41c55abd7ccc6baa3`).